iot/iot.rb

```diff
@@ -387,15 +387,15 @@
   )
 
   puts "Devices:"
-  if response.devices && response.devices.any?
+  if response.devices&.any?
```

Comment:
Is the following `response.devices&.any?`  common convention now? I find the expanded version easier to parse.

Reply:
I think the convention is fairly common. Both default rubocop and standardrb will fail if you don't use the safe navigation operator.

Reply:
It's considered idiomatic now—although it is newer syntax, and yes it is harder to parse, especially for the old-timers like me. I think it's fine to keep it, especially since it is the rubocop default. (I'm not sold on standard, but it's probably just because I'm wary about there being 15.)

Reply:
Oof. Fellow old-timer here: I do not care for that syntax.

I also feel pretty strongly that code in docs and samples should be very slow to adopt new language features. These things are adopted slowly, and it takes some people a very long time to learn about them. The fact that we're talking about whether it's common 3.5 years after its release is evidence of this.

Given the choice between an older style (`response.devices && response.devices.any?`) and a newer style (`response.devices&.any?`) I think we should err on the side of the older style until the newer style is ubiquitous.

Reply:
I enjoy `response.devices&.any?` in my codez.
I prefer `response.devices && response.devices.any?` on the website for readability.